### ISSUE TITLE
Problem z publikacją ofert - raz działa a raz nie pomimo wysyłania tych samych danych

### ISSUE THREAD

**Issue by DamianGosciewski** [allegro/allegro-api#9206]
Występuje problem: 
 - [x] z zasobem REST API / with a REST API resource
 - [x] ze środowiskiem produkcyjnym / with the production environment
 - [ ] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

:beetle: **Opis / Description**
Witam, problem dotyczy publikacji ofert poprzez bramkę `/sale/offer-publication-commands`.

Co jakiś czas wywala błąd `A product set must contain at least 1 product.` 
![obraz](https://github.com/allegro/allegro-api/assets/53616157/68bd7b48-d6ee-4e10-8c31-6ede643eaa1a)

Oferta w czasie publikacji posiada przypisany produkt.
![obraz](https://github.com/allegro/allegro-api/assets/53616157/8735c819-780c-42fc-9166-629e2f713e76)

Co ciekawe, przy kolejnej próbie publikacji (oczywiście bez zmian danych oferty) przechodzi pomyślnie.

Dane wykorzystane w przykładzie problemu:
**Numer oferty:** 15825919344
**Przypisany produkt:** 005e9908-c6ae-4297-82ee-6515fec04871
**CommandId publikacji z błędem:** 83780ee4-bbe1-45fa-913c-479d4c3f9778

**Comment by MartaNowaczyk** [comment]
Weryfikujemy opisaną sytuację, wrócę z odpowiedzią.

**Comment by MartaNowaczyk** [comment]
Okazało się, że problemem był związany z podpiętym zestawem produktowym. Z tego co widzimy udało się później aktywować tę ofertę.

**Comment by DamianGosciewski** [comment]
> Okazało się, że problemem był związany z podpiętym zestawem produktowym. Z tego co widzimy udało się później aktywować tę ofertę.

Hmmm, dziękuję za odpowiedź, jednak nie rozumiem. Jaki problem jest z podpiętym zestawem produktowym? Cały czas jest podpięty ten sam i oferty (15825919344) nie udało się aktywować. Nadal wisi jako szkic.

**Comment by MartaNowaczyk** [comment]
Ok, rozumiem, w takim razie mam nieprawidłowe informacje. Sprawdzimy to jeszcze w takim razie.

**Comment by DamianGosciewski** [comment]
Jeszcze doprecyzuję. Wskazanej oferty (15825919344) po wystąpieniu tego błędu nie da się aktywować, pomimo że dane są poprawne.

Z kolei jeżeli wystawi się nowy szkic z identycznymi danymi, tym samym przypisanym produktem, to oferta aktywuje się normalnie, bez błędu `A product set must contain at least  1 product.`

**Przykład takiej oferty z tymi samymi danymi i pomyślną aktywacją:** 15840941632.

Taki problem występuje u nas często.

**Comment by MartaNowaczyk** [comment]
Dziękuję za dodatkowe informacje. Sprawdzamy to jeszcze.

**Comment by DamianGosciewski** [comment]
> Dziękuję za dodatkowe informacje. Sprawdzamy to jeszcze.

Czy może coś już wiadomo? Może potrzeba jakiś dodatkowych informacji z naszej strony?
Sprawa jest dla nas priorytetowa, przekłada się na nasze wyniki biznesowe.

**Comment by AureliuszBrussy** [comment]
Nie, nie mam jeszcze ostatecznych ustaleń Proszę o jeszcze trochę cierpliwości. Na ten moment nie potrzebujemy dodatkowych informacji. Gdy tylko to rozstrzygniemy - damy znać.

**Comment by aszejda** [comment]
Dzień dobry. Czy udało się już odnaleźć przyczynę błędu?

**Comment by MartaNowaczyk** [comment]
Na ten moment znaleźliśmy błąd w naszym działaniu, stąd będziemy musieli wdrożyć poprawkę. Na pewno chwilę nam to zajmie, bo występuje tu sporo zależności między ofertą, produktem, a zestawem. Wrócę z informacją jak tylko uda nam się to poprawić.

**Comment by aszejda** [comment]
Bardzo dziękuję za odpowiedź. Czy jest szansa na oszacowanie "chwili"? Jest to raczej kwestia dni czy tygodni? Nie wiemy jakie decyzje podjąć na czas wprowadzania poprawek.

**Comment by MartaNowaczyk** [comment]
Myślę, że tydzień na pewno. Dokładniej ciężko mi oszacować, bo jest tu wiele zależności, które mogą wydłużyć ten czas.

**Comment by DamianGosciewski** [comment]
Wątek jak najbardziej aktualny, czekamy na informacje.

**Comment by aszejda** [comment]
Dzień dobry, Czy możemy liczyć na wprowadzenie poprawki.

**Comment by AureliuszBrussy** [comment]
Dostałem informację, że poprawka została już wdrożona.

**Comment by aszejda** [comment]
Dziękuję za informację. Faktycznie błąd już przestał występować. Pozdrawiam
